README.rst

```diff
@@ -56,62 +52,57 @@ We currently support the following runners:
 Installer scripts
 =================
 
-Lutris automates installation of games using configuration scripts written in
-JSON or YAML, which list the various files needed to install a game and can
-perform a series of actions on them.
-The syntax of installers is described in ``docs/installers.rst``, and is also
-available on `lutris.net <https://lutris.net>`_ when writing installers.
+Lutris installations are fully automated through runners, which can be written
+in either JSON or YAML.
+The runner syntax is described in ``docs/installers.rst``, and is also
+available online at `lutris.net <https://lutris.net>`_.
 
-A web UI is planned to ease the creation of these scripts.
+A web UI is planned to ease the creation of runners.
```

Comment:
I was like… wth, I never said that!… but then I realized you changed scripts to runners :laughing: